changelog.d/README.md

```diff
@@ -62,19 +62,17 @@ Do your best and a maintainer can discuss this with you during review.
 
 ### Examples
 
-Example bugfix, `2082.bugfix.md`:
+Example bugfix, [`2223.bugfix.md`](https://github.com/jazzband/pip-tools/pull/2224):
 
 ```md
-Fixed the ordering of format controls to preserve underlying `pip` behavior
--- by {user}`sethmlarson`.
+Fixed a bug which removed slashes from URLs in ``-r`` and ``-c`` in the output
+of ``pip-compile`` -- by {user}`sirosen`.
```

Comment:
@jayaddison the double backticks should've been fixed here. We missed this in review.